packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -944,9 +944,9 @@ def check_allowed_tx_type(self, transaction: Transaction) -> None:
     @classmethod
     def check_majority_possible_with_new_voter(
         cls,
-        votes_by_participant: Dict[Any, Any],
-        new_voter: Any,
-        new_vote: Any,
+        votes_by_participant: Dict[str, BaseTxPayload],
+        new_voter: str,
+        new_vote: BaseTxPayload,
         nb_participants: int,
         exception_cls: Type[ABCIAppException] = ABCIAppException,
     ) -> None:
```

Comment:
added specific types here and in matching types in `check_majority_possible` and `is_majority_possible`